projects/packages/phpcs-filter/composer.json

```diff
@@ -19,7 +19,8 @@
 	},
 	"require-dev": {
 		"automattic/jetpack-changelogger": "@dev",
-		"yoast/phpunit-polyfills": "^1.1.1"
+		"yoast/phpunit-polyfills": "^1.1.1",
```

Comment:
I assume this'll eventually need to be bumped? (Yes, it's an unnecessary comment, but I'm trying to stay engaged as I look at 400+ files...)

Reply:
Sorry for all the files. I could have split this one up more, I suppose.

Yeah, bumping this is going to be in the next PR, unless I divide it further (this one was already split out of it 😀)

Reply:
No worries! Sometimes it's easier to knock it out in one group.